.github/CODEOWNERS

```
@@ -20,6 +20,7 @@
 /network_security/intercept/  @terraform-google-modules/pm2-team @terraform-google-modules/terraform-samples-reviewers
 /network_security/mirroring/  @terraform-google-modules/pm2-team @terraform-google-modules/terraform-samples-reviewers
 /privateca/                   @terraform-google-modules/dee-infra @terraform-google-modules/terraform-samples-reviewers
+/securitycenter/          @terraform-google-modules/terraform-samples-reviewers
```

```suggestion
/securitycenter/          @terraform-google-modules/gcp-security-command-center @terraform-google-modules/terraform-samples-reviewers
```

Created in b/417061262#comment5